### ISSUE TITLE
This isn't working 

### ISSUE THREAD

**Issue by UltraThyme42** [Learningwithsebby/Making-a-working-old-version-of-prodigy#1]
http://[https-learningwithsebby.github.io-](http://https-learningwithsebby.github.io-/)/

**Comment by Learningwithsebby** [comment]
I don’t think this is working because I’m new to coding on GitHub and also new to coding so I don’t know how to do some things and also I’m doing all of this on my phone so this is hard and also thanks for inspiring me.
